Code won't compile following recent breaking changes in the starling package

Following recent breaking changes to the `starling` package, the `starling-cli` will no longer compile.

```plain
# github.com/billglover/starling-cli/cmd
cmd/listContacts.go:33:9: invalid indirect of cons (type []starling.Contact)
cmd/listContacts.go:38:17: invalid indirect of cons (type []starling.Contact)
cmd/listContacts.go:39:15: invalid indirect of cons (type []starling.Contact)
cmd/listContacts.go:47:9: invalid indirect of cons (type []starling.Contact)
cmd/listContacts.go:53:9: invalid indirect of cons (type []starling.Contact)
cmd/listContacts.go:58:17: invalid indirect of cons (type []starling.Contact)
cmd/listContacts.go:60:48: invalid indirect of cons (type []starling.Contact)
cmd/listGoals.go:33:9: invalid indirect of goals (type []starling.SavingsGoal)
cmd/listGoals.go:38:17: invalid indirect of goals (type []starling.SavingsGoal)
cmd/listGoals.go:39:15: invalid indirect of goals (type []starling.SavingsGoal)
cmd/listGoals.go:39:15: too many errors
```

Many of these changes relate to billglover/starling/issues/5 which included changes to the return types of function signatures.

--- a/cmd/listContacts.go
+++ b/cmd/listContacts.go
@@ -30,13 +30,13 @@ func listContacts(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if len(*cons) == 0 {
+	if len(cons) == 0 {
 		return
 	}
 
 	limit := viper.GetInt("limit")
-	if limit > len(*cons) {
-		limit = len(*cons)
+	if limit > len(cons) {
+		limit = len(cons)
 	}
 
 	uuid := viper.GetBool("uuid")
@@ -44,19 +44,19 @@ func listContacts(cmd *cobra.Command, args []string) {
 	if uuid == true {
 		color.Green("%3s %30s %40s\n", "#", "Name", "UUID")
 		for i := 0; i < limit; i++ {
-			c := (*cons)[i]
+			c := cons[i]
 			fmt.Printf("%s %30s %40s\n", color.BlueString("%03d", i), c.Name, c.UID)
 		}
 	} else {
 		color.Green("%3s %30s\n", "#", "Name")
 		for i := 0; i < limit; i++ {
-			c := (*cons)[i]
+			c := cons[i]
 			fmt.Printf("%s %30s\n", color.BlueString("%03d", i), c.Name)
 		}
 	}
 
-	if limit < len(*cons) {
+	if limit < len(cons) {
 		color.Set(color.FgHiMagenta)
-		fmt.Printf("%d of %d contacts\n", limit, len(*cons))
+		fmt.Printf("%d of %d contacts\n", limit, len(cons))
 	}
 }

--- a/cmd/listGoals.go
+++ b/cmd/listGoals.go
@@ -30,13 +30,13 @@ func listGoals(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if len(*goals) == 0 {
+	if len(goals) == 0 {
 		return
 	}
 
 	limit := viper.GetInt("limit")
-	if limit > len(*goals) {
-		limit = len(*goals)
+	if limit > len(goals) {
+		limit = len(goals)
 	}
 
 	uuid := viper.GetBool("uuid")
@@ -44,7 +44,7 @@ func listGoals(cmd *cobra.Command, args []string) {
 	if uuid == true {
 		color.Green("%s %-20s %10s %10s %11s %40s\n", "  #", "Name", "Saved", "Target", "Percentage", "UID")
 		for i := 0; i < limit; i++ {
-			g := (*goals)[i]
+			g := goals[i]
 			saved := float64(g.TotalSaved.MinorUnits) / 100
 			target := float64(g.Target.MinorUnits) / 100
 			fmt.Printf("%s %-20s %10.2f %10.2f %10d%% %40s\n", color.BlueString("%03d", i), g.Name, saved, target, g.SavedPercentage, g.UID)
@@ -52,15 +52,15 @@ func listGoals(cmd *cobra.Command, args []string) {
 	} else {
 		color.Green("%s %-20s %10s %10s %11s\n", "  #", "Name", "Saved", "Target", "Percentage")
 		for i := 0; i < limit; i++ {
-			g := (*goals)[i]
+			g := goals[i]
 			saved := float64(g.TotalSaved.MinorUnits) / 100
 			target := float64(g.Target.MinorUnits) / 100
 			fmt.Printf("%s %-20s %10.2f %10.2f %10d%%\n", color.BlueString("%03d", i), g.Name, saved, target, g.SavedPercentage)
 		}
 	}
 
-	if limit < len(*goals) {
+	if limit < len(goals) {
 		color.Set(color.FgHiMagenta)
-		fmt.Printf("%d of %d savings goals\n", limit, len(*goals))
+		fmt.Printf("%d of %d savings goals\n", limit, len(goals))
 	}
 }

--- a/cmd/listTransactions.go
+++ b/cmd/listTransactions.go
@@ -31,13 +31,13 @@ func listTransactions(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if len(*txns) == 0 {
+	if len(txns) == 0 {
 		return
 	}
 
 	limit := viper.GetInt("limit")
-	if limit > len(*txns) {
-		limit = len(*txns)
+	if limit > len(txns) {
+		limit = len(txns)
 	}
 
 	uuid := viper.GetBool("uuid")
@@ -45,19 +45,19 @@ func listTransactions(cmd *cobra.Command, args []string) {
 	if uuid == true {
 		color.Green("%-3s %-24s %-10s %-30s %-40s\n", "#", "Created", "Amount", "Narrative", "UUID")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %-24s %10.2f %-30s %-40s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative, txn.UID)
 		}
 	} else {
 		color.Green("%-3s %-24s %-10s %-30s\n", "#", "Created", "Amount", "Narrative")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %-24s %10.2f %-30s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative)
 		}
 	}
 
-	if limit < len(*txns) {
+	if limit < len(txns) {
 		color.Set(color.FgHiMagenta)
-		fmt.Printf("%d of %d transactions\n", limit, len(*txns))
+		fmt.Printf("%d of %d transactions\n", limit, len(txns))
 	}
 }

--- a/cmd/listTransactionsCard.go
+++ b/cmd/listTransactionsCard.go
@@ -31,13 +31,13 @@ func listTransactionsCard(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if len(*txns) == 0 {
+	if len(txns) == 0 {
 		return
 	}
 
 	limit := viper.GetInt("limit")
-	if limit > len(*txns) {
-		limit = len(*txns)
+	if limit > len(txns) {
+		limit = len(txns)
 	}
 
 	uuid := viper.GetBool("uuid")
@@ -45,19 +45,19 @@ func listTransactionsCard(cmd *cobra.Command, args []string) {
 	if uuid == true {
 		color.Green("%3s %30s %10s %30s %40s\n", "#", "Created", "Amount", "Narrative", "UUID")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s %40s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative, txn.UID)
 		}
 	} else {
 		color.Green("%3s %30s %10s %30s\n", "#", "Created", "Amount", "Narrative")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative)
 		}
 	}
 
-	if limit < len(*txns) {
+	if limit < len(txns) {
 		color.Set(color.FgHiMagenta)
-		fmt.Printf("%d of %d transactions\n", limit, len(*txns))
+		fmt.Printf("%d of %d transactions\n", limit, len(txns))
 	}
 }

--- a/cmd/listTransactionsDD.go
+++ b/cmd/listTransactionsDD.go
@@ -31,13 +31,13 @@ func listTransactionsDD(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if len(*txns) == 0 {
+	if len(txns) == 0 {
 		return
 	}
 
 	limit := viper.GetInt("limit")
-	if limit > len(*txns) {
-		limit = len(*txns)
+	if limit > len(txns) {
+		limit = len(txns)
 	}
 
 	uuid := viper.GetBool("uuid")
@@ -45,19 +45,19 @@ func listTransactionsDD(cmd *cobra.Command, args []string) {
 	if uuid == true {
 		color.Green("%3s %30s %10s %30s %40s\n", "#", "Created", "Amount", "Narrative", "UUID")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s %40s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative, txn.UID)
 		}
 	} else {
 		color.Green("%3s %30s %10s %30s\n", "#", "Created", "Amount", "Narrative")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative)
 		}
 	}
 
-	if limit < len(*txns) {
+	if limit < len(txns) {
 		color.Set(color.FgHiMagenta)
-		fmt.Printf("%d of %d transactions\n", limit, len(*txns))
+		fmt.Printf("%d of %d transactions\n", limit, len(txns))
 	}
 }

--- a/cmd/listTransactionsFPSIn.go
+++ b/cmd/listTransactionsFPSIn.go
@@ -31,13 +31,13 @@ func listTransactionsFPSIn(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if len(*txns) == 0 {
+	if len(txns) == 0 {
 		return
 	}
 
 	limit := viper.GetInt("limit")
-	if limit > len(*txns) {
-		limit = len(*txns)
+	if limit > len(txns) {
+		limit = len(txns)
 	}
 
 	uuid := viper.GetBool("uuid")
@@ -45,19 +45,19 @@ func listTransactionsFPSIn(cmd *cobra.Command, args []string) {
 	if uuid == true {
 		color.Green("%3s %30s %10s %30s %40s\n", "#", "Created", "Amount", "Narrative", "UUID")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s %40s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative, txn.UID)
 		}
 	} else {
 		color.Green("%3s %30s %10s %30s\n", "#", "Created", "Amount", "Narrative")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative)
 		}
 	}
 
-	if limit < len(*txns) {
+	if limit < len(txns) {
 		color.Set(color.FgHiMagenta)
-		fmt.Printf("%d of %d transactions\n", limit, len(*txns))
+		fmt.Printf("%d of %d transactions\n", limit, len(txns))
 	}
 }

--- a/cmd/listTransactionsFPSOut.go
+++ b/cmd/listTransactionsFPSOut.go
@@ -31,13 +31,13 @@ func listTransactionsFPSOut(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if len(*txns) == 0 {
+	if len(txns) == 0 {
 		return
 	}
 
 	limit := viper.GetInt("limit")
-	if limit > len(*txns) {
-		limit = len(*txns)
+	if limit > len(txns) {
+		limit = len(txns)
 	}
 
 	uuid := viper.GetBool("uuid")
@@ -45,19 +45,19 @@ func listTransactionsFPSOut(cmd *cobra.Command, args []string) {
 	if uuid == true {
 		color.Green("%3s %30s %10s %30s %40s\n", "#", "Created", "Amount", "Narrative", "UUID")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s %40s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative, txn.UID)
 		}
 	} else {
 		color.Green("%3s %30s %10s %30s\n", "#", "Created", "Amount", "Narrative")
 		for i := 0; i < limit; i++ {
-			txn := (*txns)[i]
+			txn := txns[i]
 			fmt.Printf("%s %30s %10.2f %30s\n", color.BlueString("%03d", i), txn.Created, txn.Amount, txn.Narrative)
 		}
 	}
 
-	if limit < len(*txns) {
+	if limit < len(txns) {
 		color.Set(color.FgHiMagenta)
-		fmt.Printf("%d of %d transactions\n", limit, len(*txns))
+		fmt.Printf("%d of %d transactions\n", limit, len(txns))
 	}
 }

